source/brailleDisplayDrivers/seika.py

```diff
@@ -96,18 +96,19 @@ def terminate(self):
 			self._ser.close()
 
 	def display(self, cells: List[int]):
+
+		# add padding so total length is 1 + numberOfStatusCells + numberOfRegularCells
+		cellPadding: bytes = bytes(self.numCells - len(cells))
+		writeBytes: List[bytes] = [self.s40, ]
 		# every transmitted line consists of the preamble SEIKA_SENDHEADER and the Cells
 		if self.numCells==80:
-			lineBytes = b"".join([
-				b"\xff\xff\x73\x38\x30\x00\x00\x00",
-				bytes(cells)
-			])
+			lineBytes: bytes = self.s40 + bytes(cells) + cellPadding
 		else:
-			lineBytes = b"".join([
-				self.s40,
-				b"\0",
-				bytes(cells)
-			])
+			for cell in cells:
+				writeBytes.append(b"\x00")
+				writeBytes.append(intToByte(cell))
+			lineBytes = b"".join(writeBytes)
+			# + cellPadding		self._ser.write(lineBytes)

```

Comment:
This seems like it was accidentally left in.

Reply:
There need to reach 40/80 bytes because the Seika device is waiting for reading the 40/80 number of the bytes at a time.

Reply:
This line is commented out, I assume it should be deleted.